bin/tests/integration/named_metrics_tests.rs

```diff
@@ -158,6 +165,141 @@ fn test_prometheus_endpoint_startup() {
     })
 }
 
+#[test]
```

Comment:
Squash this into the previous commit adding this file.

Reply:
Done.